metrics/windows/filesystem.go

```diff
@@ -13,33 +13,35 @@ import (
 
 // FilesystemGenerator XXX
 type FilesystemGenerator struct {
+	IgnoreRegexp *regexp.Regexp
 }
 
 // NewFilesystemGenerator XXX
-func NewFilesystemGenerator() (*FilesystemGenerator, error) {
-	return &FilesystemGenerator{}, nil
+func NewFilesystemGenerator(ignoreReg *regexp.Regexp) (*FilesystemGenerator, error) {
```

Comment:
https://github.com/mackerelio/mackerel-agent/pull/303/commits/e8905fd0f3f4be650c32597691d754357bc6385f#diff-058fec3ec3456eaf65ea8739c6e7bad5R39 There is!!! I'm sorry :innocent: